packages/web/src/scss/components/Toast/dynamic-toast.mjs

```diff
@@ -6,8 +6,13 @@ export const addDynamicToast = (event, containerId) => {
     containerId,
     autoCloseInterval: formElement.querySelector('#toast-auto-close-interval').value,
     color: formElement.querySelector('#toast-color').value,
-    content: formElement.querySelector('#toast-content').value,
     enableAutoClose: formElement.querySelector('#toast-enable-auto-close').checked,
+    message: formElement.querySelector('#toast-message').value,
+    enableLink: formElement.querySelector('#toast-enable-link').checked,
```

Comment:
i am still not sure if we need this `enableLink` value, maybe we can do it just with`linkContent` or `linkContent && linkProps.href`